integration_tests/suite/test_user_room.py

```diff
@@ -231,6 +232,30 @@ def test_create_minimal_parameters(self):
         )
         self._delete_room(room)
 
+    @fixtures.http.room(users=[USER_1, USER_2, USER_3])
+    def test_create_when_a_group_of_users_already_exists(self, existing_room):
+        room_args = {
+            'users': [USER_1, USER_2],
+        }
+        room = self.chatd.rooms.create_from_user(room_args)
+
+        assert_that(room['uuid'], not_(existing_room['uuid']))
+        self._delete_room(room)
```

Comment:
why the `self._delete_room`? Isn't it part of the fixture teardown?

Reply:
This call delete the room from the `self.chatd.rooms.create_from_user` call, not from the fixture

Reply:
ah oops, my bad